wiki/Community/Discord_servers/en.md

```diff
@@ -1,9 +1,105 @@
+| [osu! Medal Hunters](https://discord.gg/osumedalhunters) | [MegaMix](https://osu.ppy.sh/users/18152711) | **osu! Medal Hunters** hosts discussion about [medals](/wiki/Medals) and [medal solutions](/wiki/Medals/Unlock_requirements). |
+| [osu!alternative](https://discord.gg/VZWRZZXcW4) | [billie eilish](https://osu.ppy.sh/users/6245906) | **osu!alternative** offers extra metrics and score trackers not provided by the website. It allows players to rank players and scores in an unofficial manner. |
+
+## Community development
+
+These servers focus on providing and developing tools and systems for the game. While the official development of osu! is done in the **osu!** Discord server, certain projects require their own workspaces before proposing and merging their work in game.
+
+| Name | Owner |
+| :-- | :-- |
+| [Performance Points](https://discord.gg/aqPCnXu) | - |
+| [osu!catch dev server](https://discord.gg/YEJBENvFzN) | [bastoo0](https://osu.ppy.sh/users/4864877) |
+| [o!m SR/PP Rework Hub](https://discord.gg/GFCNNg8bwk) | [Quenlla](https://osu.ppy.sh/users/4725379) |
+
+## Mapping and modding
+
+The **Modding & Mapping Hubs** are community Discord servers designed for all current and aspiring mappers and modders. These are places meant to interact between other users interested in mapping for the game mode. They offer a place to organise, discuss, and advertise different mapping projects.
+
+| Name | Owner |
+| :-- | :-- |
+| [osu!taiko Modding & Mapping Hub](https://discord.gg/yRjvvyZ) | [Hivie](https://osu.ppy.sh/users/14102976) |
+| [osu!catch Modding and Mapping Hub](https://discord.gg/ZuxFc4q) | [Ascendance](https://osu.ppy.sh/users/2931883) |
+| [osu!mania Mapping & Modding Hub](https://discord.gg/FqbDdYN) | [Maxus](https://osu.ppy.sh/users/4335785) |
+
+### Other
+
+Besides the game mode hubs, other community servers exist to facilitate communication between mappers, modders, and other users alike.
+
+| Name | Owner | Description |
+| :-- | :-- | :-- |
+| [Mapset Management Server](https://discord.gg/TCDSjhb6yS) | [Kyuunex](https://osu.ppy.sh/users/9236044) | The **Mapset Management Server** hosts spaces to create and browse requests for [mods](/wiki/Modding), [guest difficulties](/wiki/Beatmap/Guest_difficulty), and [collaborations](/wiki/Beatmap/Beatmap_collaborations) in a casual environment. |
+| [Cozy Mappers' Café](https://discord.gg/uDjwjaZ) | [Teky](https://osu.ppy.sh/users/10520912) | **Cozy Mappers' Café** aims to be a laid back community, where users can discuss all things mapping and modding related. |
+
+## Tournaments and contests
+
+These servers aim to provide users everything related to [tournaments](/wiki/Tournaments) and [contests](/wiki/Contests). By providing users with regular notices on upcoming tournaments, users can stay tuned with the latest tournaments they can be a part of. Both players and staff members can benefit from the resources provided by these servers.
```

Comment:
probably dont mention contests cuz theres nothing abt that here

Reply:
Not yet. Was planning ahead in advance. Though if we want to stick with what we have now, it can be removed.

In the case contest-specific servers were added though, they should go together imo.

Reply:
i agree w/ the last part but yeah for now there are no contests so i dont think there's any reason to mention it